### DISCUSSION TITLE
Show file and folder "." typically ".htaccess"

### DISCUSSION THREAD

**Discussion by WebSite21** [Natizyskunk/vscode-sftp#71] [Q&A] [answered]
Hello, I am trying to use SFTP extension via SFTP exolorer to display files and folders starting with "." typically .htacess, etc. I can't find how to set / influence this behavior. Is it possible to influence this with sftp.json?

Have nic a day

**Comment by Natizyskunk** [comment, marked as answer]
By default, the sftp extension should always show all your dotfiles.
If not maybe there is something to configure on your remote.
